src/Navigation.js

```diff
@@ -91,77 +92,92 @@ const HomeStack = StackNavigator(
   }
 );
 
-const EventsStack = StackNavigator(
+const EventsStack = createStackNavigator(
   {
     [EVENT_LIST]: { screen: withShadow(EventsScreen) },
     [EVENT_DETAILS]: { screen: EventDetailsScreen },
     [EVENT_CATEGORIES_FILTER]: { screen: CategoriesFilterScreen }
   },
   {
     initialRouteName: EVENT_LIST,
-    navigationOptions: {
-      tabBarIcon: tabIcon(iconEventsDefault, iconEventsActive),
-      tabBarLabel: text.tabEvents,
-      tabBarTestIDProps: {
-        testID: "events-tab-button"
-      }
-    },
     headerMode: "none"
   }
 );
 
-const ParadeStack = StackNavigator(
+const ParadeStack = createStackNavigator(
   {
-    [PARADE]: { screen: withShadow(ParadeInformationScreen) }
+    [PARADE]: { screen: withShadow(() => <View />) }
```

Comment:
This line needs to stay the same for the ParadeInformationScreen to show. (It was added while you were developing this)

Reply:
Also done!